Makefile

```diff
@@ -7,12 +7,18 @@ SKOPEO_BRANCH = master
 SUDO =
 
 ifeq ($(shell uname),Darwin)
-DARWIN_BUILD_TAG = containers_image_ostree_stub
+PLATFORM_BUILD_TAG = containers_image_ostree_stub containers_image_openpgp
 endif
-BUILDTAGS = btrfs_noversion libdm_no_deferred_remove $(DARWIN_BUILD_TAG)
+ifeq ($(GOOS),darwin)
+PLATFORM_BUILD_TAG = containers_image_ostree_stub containers_image_openpgp
+endif
+ifeq ($(GOOS),windows)
+PLATFORM_BUILD_TAG = containers_image_ostree_stub containers_image_openpgp
+endif
+BUILDTAGS = btrfs_noversion libdm_no_deferred_remove $(PLATFORM_BUILD_TAG)
 BUILDFLAGS := -tags "$(BUILDTAGS)"
 
-PACKAGES := $(shell go list $(BUILDFLAGS) ./... | grep -v github.com/containers/image/vendor)
+PACKAGES ?= $(shell go list $(BUILDFLAGS) ./... | grep -v github.com/containers/image/vendor)
```

Comment:
Why is this necessary?

Reply:
oh it was when i was going to set `PACKAGES` earlier in the makefile, but didn't need to as the build tags filtered the packages that were causing issues.